client/dashboard/sites/staging-site-delete-modal/index.tsx

```diff
@@ -49,7 +49,10 @@ export default function StagingSiteDeleteModal( {
 			onSuccess: () => {
 				recordTracksEvent( 'calypso_hosting_configuration_staging_site_delete_success' );
 				if ( window?.location?.pathname?.startsWith( '/v2' ) ) {
-					createSuccessNotice( __( 'Staging site deleted.' ), { type: 'snackbar' } );
+					createSuccessNotice(
+						__( 'We are deleting your staging site. We will send you an email when it is done.' ),
```

Comment:
Since we are not sending an email on staging site deletion, we could do something like

```suggestion
						__( 'We are deleting your staging site. We will let you know when it is done.' ),
```

Reply:
Agreed, this wording or the `We will notify you when it's done` work for me

Reply:
adjusted in https://github.com/Automattic/wp-calypso/pull/105671/commits/71bf8a14bcbaa54cd138fe999a45e3a67a4c1999